nibabel/loadsave.py

```diff
@@ -212,43 +203,3 @@ def read_img_data(img, prefer='scaled'):
         if prefer == 'scaled':
             return hdr.data_from_fileobj(fileobj)
         return hdr.raw_data_from_fileobj(fileobj)
-
-
-def which_analyze_type(binaryblock):
```

Comment:
Sorry man - I know this is anal - but this one's part of the API too, technically.